R/extract_parameters.R

```diff
@@ -229,6 +206,26 @@
   }
 
 
+
+  # ==== Std Coefficients for other methods than "refit"
+
+  if (!is.null(standardize)) {
+    # give minimal attributes required for standardization
+    temp_pars <- parameters
+    class(temp_pars) <- c("parameters_model", class(temp_pars))
+    attr(temp_pars, "ci") <- ci
+    attr(temp_pars, "obj_name") <- model # pass the model as is (this is a cheat - teehee!)
+
+    std_parms <- effectsize::standardize_parameters(temp_pars, method = standardize)
+    parameters$Coefficient <- std_parms$Std_Coefficient
+    parameters$SE <- attr(std_parms, "standard_error")
+
+    if (!is.null(ci)) {
+      parameters$CI_low <- std_parms$CI_low
```

Comment:
Should be "Std_CI_low" as name (and for high as well)

Reply:
These were previously just `CI_low` / `CI_high`, without the `Std_` prefix.